jaxrs/src/main/java/com/cerner/beadledom/jaxrs/exceptionmappers/WebApplicationExceptionMapper.java

```diff
@@ -0,0 +1,69 @@
+package com.cerner.beadledom.jaxrs.exceptionmappers;
+
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+import com.cerner.beadledom.jaxrs.models.JsonError;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A Mapper for {@link WebApplicationException}s that sets the {@code Content-Type} header and
```

Comment:
These first sentence descriptions for each of the exception mappers should be consistent. It should definitely be called out as an `Exception mapper` or `{@link ExceptionMapper}` rather than just a `Mapper`.

Reply:
Updated each of these to say `An {@link ExceptionMapper} for the {@link [enter exception class here]} family of exceptions.`: https://github.com/cerner/beadledom/pull/45/commits/865a017a55ce93a2c50c539260b17f6bbcdfe110#diff-94f0fc074c60a4a4d74f16bebf75afc9R17